CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 2020.5.1
+
+## ML
+* Add l2 and validation_split to BertClassifier
+
```

Comment:
I think you forgot some changes you made to semantic similarity

Reply:
👍 added